citrus-remote-maven-plugin/pom.xml

```diff
@@ -1,11 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <artifactId>citrus-remote</artifactId>
-    <groupId>com.consol.citrus</groupId>
-    <version>3.5.0-SNAPSHOT</version>
+    <groupId>org.citrusframework</groupId>
+    <version>4.3.0-SNAPSHOT</version>
```

Comment:
We should set this to `4.3.1.-SNAPSHOT`

Reply:
because you want to sync it with citrus? I think that's a bad idea. what if there's a bugfix in citrus-remote, but not in citrus. versions get out of sync so easily. I think it should just be `4.0.0` (and `-SNAPSHOT`).

Reply:
@christophd I think you'd agree here (before merging)

Reply:
I think we should try to keep in sync and yes it may be tedious. I would try to only differ in minor releases when there is a bugfix. As a long-term objective we should bring citrus-remote back to citrus core project

Reply:
ok, if that's the goal :)